atom/browser/api/atom_api_web_contents.cc

```diff
@@ -316,7 +316,8 @@ WebContents::WebContents(v8::Isolate* isolate,
     : content::WebContentsObserver(web_contents), type_(type) {
   const mate::Dictionary options = mate::Dictionary::CreateEmpty(isolate);
   if (type == REMOTE) {
-    web_contents->SetUserAgentOverride(GetBrowserContext()->GetUserAgent());
+    web_contents->SetUserAgentOverride(GetBrowserContext()->GetUserAgent(),
+                                       false);
```

Comment:
Does this `false` mean that we won't override the UA after a navigation?

Reply:
@MarshallOfSound ?